src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -95,5 +100,61 @@ private List<String> getEmailAddress(List<Element<PartyDetails>> partyDetails) {
     }
 
 
+    public void sendEmailToCafcass(CaseDetails caseDetails) {
```

Comment:
Please rename this method to some other logical name, as this covers for both CAFCASS and Other Parties emails.